types/signer.go

```diff
@@ -53,4 +56,6 @@ type KeyManager interface {
 	AddShare(shareKey *bls.SecretKey) error
 	// RemoveShare removes a share key
 	RemoveShare(pubKey string) error
+	// AddNetworkKey saves a network key
+	AddNetworkKey(sk *rsa.PrivateKey) error
```

Comment:
So after discussing with @y0sher and @olegshmuelov

Get rid of the keyManager interface. `Share` functions should move to `BeaconSigner`. `Network` (rename to SSV) functions should go to SSVSigner

The `KeyManager` in testingutils can implement both to reduce changes (but you can do some other design if you wish)